pkg/adapter/runtime.go

```diff
@@ -94,6 +99,14 @@ func (r *LocalRuntime) GetRWImages() ([]*ContainerImage, error) {
 	return r.getImages(true)
 }
 
+func (r *LocalRuntime) getImage(imageID string) (*image.Image, error) {
+	image, err := r.Runtime.ImageRuntime().GetImage(imageID)
```

Comment:
Just return r.Runtime.ImageRuntime().GetImage(imageID)

Reply:
Actually looking at it further, I don't need this function and don't call it.  Backing this change out.